QuickStart.md

```diff
@@ -17,13 +17,17 @@ kubectl create namespace bring-your-own-datacenter
 
 ![](images/ByoDC-Quickstart-1-Namespace.png)
 
-## 3.) Add Helm Repo
+## 3.) Install Emissary Ingress
+
+Install Emissary Ingress v3.x using the [official installation instructions](https://www.getambassador.io/docs/emissary/latest/topics/install/helm).
```

Comment:
Maybe we can pin the link to 3.x? E.g. `https://www.getambassador.io/docs/emissary/3.5/topics/install/helm`

Reply:
good idea 🙏